backend/internal/handlers/organizations/organization.go

```diff
@@ -123,3 +123,41 @@ func (service *OrganizationService) GetClassroomsInOrg() fiber.Handler {
 		})
 	}
 }
+
+func (service *OrganizationService) GetOrgTemplateRepos() fiber.Handler {
+	return func(c *fiber.Ctx) error {
+		// Extract org_id from the path
+		orgName := c.Params("org_name")
+		if orgName == "" || orgName == "undefined" {
+			log.Default().Println("Error getting org_name: ", orgName)
+			return c.Status(400).JSON(fiber.Map{"error": "invalid org_name"})
+		}
+
+		// Parse pagination parameters
+		itemsPerPage := c.QueryInt("itemsPerPage")
```

Comment:
QueryInt defaults to 0 or 1 for no value or a non integer respectively. Doing something like this
itemsPerPage, err = strconv.ParseInt(c.Params("itemsPerPage"), 10, 64)
Allows us to catch those cases as errors, rather than potentially returning something unexpected